toolbox/tests/reminder_batch_scheduler/test_reminder_batch.py

```diff
@@ -164,18 +164,16 @@ def test_build_batch_count_query(batch, wave_classifiers, expected_query, expect
         'P_RL_1RL1_1': "case_type != 'HI' AND treatment_code IN ('HH_LP1E', 'HH_LP2E')"
                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')",
         'P_RL_1RL2B_1': "case_type != 'HI' AND treatment_code IN ('HH_LP1W', 'HH_LP2W')"
-                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')"
+                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')",
+        'P_QU_H1': "case_type != 'HI' AND treatment_code IN ('HH_LP1E') AND print_batch IN ('1','2','3')",
+        'P_QU_H2': "case_type != 'HI' AND treatment_code IN ('HH_LP1W') AND print_batch IN ('1','2','3')"
```

Comment:
Same comment as above

Reply:
what does `print_batch` of `3` even mean anymore? It should be meaningless